### ISSUE TITLE
Fatal error "undefined method WP_Error::get_message()"

### ISSUE THREAD

**Issue by Screenfeed** [wp-media/imagify-plugin#375]
```
PHP Fatal error: Uncaught Error: Call to undefined method WP_Error::get_message() in /wp-content/plugins/imagify/classes/Optimization/Process/AbstractProcess.php:1111
```

The method name is `get_error_message()`.
[Source](https://wordpress.org/support/topic/php-fatal-error-171/).

**Comment by Alstria** [comment]
Getting this error too. First 20 images working fine. Now it's totally broken

**Comment by Screenfeed** [comment]
Closed in #380.
